package.json

```diff
@@ -1762,6 +1767,27 @@
           "default": false,
           "description": "Default boolean value for automatically sharing R browser ports with guests."
         },
+        "r.plot.devArgs" :{
+          "type": "object",
+          "markdownDescription": "The arguments for the png device to replay user graphics to show in VSCode. Requires `#r.plot.useHttpgd#` to be set to `false`. \n\nChanges the option `vsc.dev.args` in R.",
+          "default": {
+            "width": 800,
```

Comment:
Maybe we could follow the default png width and height or at least keep its 1:1 aspect ratio as default?

```r
png(filename = "Rplot%03d.png",
    width = 480, height = 480, units = "px", pointsize = 12,
     bg = "white",  res = NA, ...,
    type = c("cairo", "cairo-png", "Xlib", "quartz"), antialias)
```